usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1921,6 +1926,11 @@ static CK_RV import_EC_key(STDLL_TokData_t * tokdata, SESSION * sess,
         return rc;
     }
 
+    if (!ep11tok_ec_curve_supported2(tokdata, ec_key_obj->template, &curve)) {
+        TRACE_ERROR("Curve not supported.\n");
+        return CKR_CURVE_NOT_SUPPORTED;
+    }
```

Comment:
Looks good to me in general. Maybe a small trace enhancement: can you include the curve NID into this error message? A textual representation would be even better, but I think we don't have one here.

Reply:
Well, I don't always have the NID of the curve here. I only have the OID in the template, and thats hard to print and read as a human. On a negative return of ep11tok_ec_curve_supported2 the returned curve parameter may be NULL, so I can't always access the curve details.